__tests__/app.test.js

```diff
@@ -101,4 +101,27 @@ describe("/api/articles", () => {
         });
     });
   });
+  describe("GET article comments by id", () => {
+    test("200 + returns array of comments with correct properties and values", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body.comments;
+          expect(comments).toBeSortedBy("created_at", { descending: true });
+          comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                article_id: expect.any(Number),
+              })
+            );
+          });
+        });
+    });
+  });
```

Comment:
Good stuff so far - but we need some more testing and error handling!
Think of the following situations...
What if there was a valid article with no comments? (status 200)
What if the request is valid, but the articleID isnt found? (status 404)
And finally, an invalid articleID in the request like (/api/articles/banana/comments)  (status 400)

Feel free to look at the notes on error handling notes to help with this!